resources/dicts/clan_symbols.json

```diff
@@ -677,7 +677,7 @@
   },
   "Fire": {
     "variants": 0,
```

Comment:
you added tags, but didn't change the variant number?

Reply:
That's because I didn't make a variant for "Fire", I just noticed that I could add some tags to it alongside "Flame" since a lot of symbols that haven't been added already had tags as well. I see how that was confusing though!